pysolarfocus/__init__.py

```diff
@@ -17,283 +16,10 @@ class Systems(str, Enum):
     
 from .modbus_wrapper import ModbusConnector
 from .component_factory import ComponentFactory
-from .components.base.component import Component
-from .components.base.data_value import DataValue
-from .components.base.enums import RegisterTypes
-from .const import SLAVE_ID, SMART_GRID_EINSCHALTUNG, SMART_GRID_NORMALBETRIEB
```

Comment:
How are we going to set states such as `SMART_GRID_EINSCHALTUNG` or `SMART_GRID_NORMALBETRIEB`?
Is the consuming application expected to handle this logic?
If so it's expected the user knows that these values exist in the `const` module, which is not optimal.
WDYT?

Reply:
Your concern applies to all properties, which are represented by a `SELECT` entity in home-assistant. I'd recommend to remove the existing methods, as they were not supporting multi-component and create an issue to provide a proper interface for them later -> #17